### PR TITLE
docs: use generated og image

### PR DESCRIPTION
**Pull request by kecrily**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Using automatically generated og image

#### Is there anything you'd like reviewers to focus on?

It is part of https://github.com/eslint/eslint.org/pull/473

### CODE REVIEW

File: docs/src/_includes/layouts/base.html
Comment (mdjermanovic) [inline docs/src/_includes/layouts/base.html:21]: I'm guessing that this shouldn't be `localhost`. Perhaps `site.hostname`?

Comment (kecrily) [reply docs/src/_includes/layouts/base.html:21]: Sorry, I was using it in testing and forgot to change it back

**Review by nzakas — APPROVED** [review]
LGTM. Would like @mdjermanovic to verify his changes before merging.

File: docs/src/_includes/layouts/base.html
Comment (mdjermanovic) [inline docs/src/_includes/layouts/base.html:21]:
```suggestion
        "https://", site.hostname, "/og",
```

**Review by nzakas — APPROVED** [review]

**Review by amareshsm — APPROVED** [review]
LGTM Waiting for Milos review.

**Comment by nzakas** [comment]
Note: https://github.com/eslint/eslint.org/pull/473 has been merged.

File: docs/src/_includes/layouts/base.html
Comment (mdjermanovic) [inline docs/src/_includes/layouts/base.html:22]: I think we should url-encode these two. For example, this wouldn't work as intended for a title that contains a `&` character, like `Configure Plugins & Parsers` (we actually had this title a few versions ago). Page descriptions are currently the same for all pages but we could change it to include rule descriptions which might also contain characters that would break the intended URL.

**Review by harish-sethuraman — APPROVED** [review]
LGTM.

As Milos mentioned the page descriptions are the same for almost all the pages. We can consider introducing this to rules page with rules description also. Rules descriptions are markdown strings so we'd need to find a way to render them.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
